packages/core/src/components/dialog/dialogStep.tsx

```diff
@@ -57,6 +57,10 @@ export interface IDialogStepProps extends Props, Omit<HTMLDivProps, "id" | "titl
      * Props for the next button.
      */
     nextButtonProps?: DialogStepButtonProps;
+    /**
+     * Disable panel for side navigation
+     */
+    panelDisabled?: boolean;
```

Comment:
I think this could simply be called `disabled`

```suggestion
    disabled?: boolean;
```